.travis.yml

```diff
@@ -20,7 +20,12 @@ env:
 
 before_install:
 - if [ "$TRAVIS_OS_NAME" == "linux" ]; then sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test; fi
+- if [ "$TRAVIS_OS_NAME" == "linux" ]; then sudo add-apt-repository -y ppa:george-edison55/precise-backports; fi
 - if [ "$TRAVIS_OS_NAME" == "linux" ]; then sudo apt-get update -qq; fi
+- if [ "$TRAVIS_OS_NAME" == "linux" ]; then sudo apt-get install -qq cmake; fi
+- if [ "$TRAVIS_OS_NAME" == "linux" ]; then mkdir cmake && travis_retry wget --quiet -O - "http://www.cmake.org/files/v3.3/cmake-3.3.1-Linux-x86_64.tar.gz" | tar --strip-components=1 -xz -C cmake && export PATH=${DEPS_DIR}/cmake/bin:${PATH}; fi
```

Comment:
seems very fragile depending on getting a specific version of cmake. why does this not work with the travis cmake? does this bump up our minimum required cmake version (which would not be desirable)?